FixtureController.php

```diff
@@ -141,11 +141,11 @@ class FixtureController extends \yii\console\controllers\FixtureController
     /**
      * @var string Alias to the template path, where all tables templates are stored.
      */
-    public $templatePath = '@tests/unit/templates/fixtures';
+    public $templatePath = '@tests/fixtures/templates';
     /**
      * @var string Alias to the fixture data path, where data files should be written.
      */
-    public $fixtureDataPath = '@tests/unit/fixtures/data';
+    public $fixtureDataPath = '@tests/fixtures/data';
```

Comment:
this would be breaking BC, can we fix overall consistency without BC break? And change the Paths in version 2.1 to be independent from unit tests then.

Reply:
We could hardcode the former `fixtureDataPath` value into the `config/console.php` configuration file of the `yii2-app-basic` template:

https://github.com/yiisoft/yii2-app-basic/blob/561b98a03943d61efece28a648142e3ac3d9596c/config/console.php#L31-L37

Like this:

```php
    'controllerMap' => [
        'fixture' => [ // Fixture generation command line.
            'class' => 'yii\faker\FixtureController',
            'fixtureDataPath' => '@tests/unit/fixtures/data',
        ],
    ],
```

Reply:
@ricpelo won't help since app templates are never updated in existing projects.

Reply:
@samdark You're right. Maybe we could do it for Yii 2.1?

Reply:
That could be great!

Reply:
@samdark May I need to create another PR, this time against 2.1 branch?

Reply:
Yes.